components/molecules/PullRequestSocialCard/pull-request-social-card.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import CardHorizontalBarChart, { LanguageObject } from "../CardHorizontalBarChart/card-horizontal-bar-chart";
+import Text from "components/atoms/Typography/text";
+import { getAvatarLink } from "lib/utils/github";
+import Avatar from "components/atoms/Avatar/avatar";
+import { IconContext } from "react-icons";
+import { VscGitPullRequest } from "react-icons/vsc";
+import { GoComment } from "react-icons/go";
+import { FaRegCommentAlt } from "react-icons/fa";
+
+interface PullRequestSocialCardProps {
+  orgName: string;
+  repoName: string;
+  languageList: LanguageObject[];
+  prTicketId: string;
+  prTitle: string;
+  commentsCount?: number;
+  avatar?: string;
+  linesAdded?: number;
+  linesRemoved?: number;
+}
+const PullRequestSocialCard = ({
+  orgName,
+  repoName,
+  prTitle,
+  avatar,
+  languageList,
+  prTicketId,
+  commentsCount,
+  linesAdded,
+  linesRemoved
+}: PullRequestSocialCardProps) => {
+  return (
+    <div className="w-[26.375rem] cursor-pointer border rounded-xl px-5 py-4">
+      <div className="flex gap-2 items-center ">
+        <Avatar alt="" avatarURL={avatar} initials="" size="sm" className="" />
```

Comment:
Remove empty className

```suggestion
        <Avatar alt="" avatarURL={avatar} initials="" size="sm"  />
```